README.md

```diff
@@ -1,6 +1,3 @@
 # AlgebraicStockFlow
-
-[![Build Status](https://travis-ci.com/Xiaoyan-Li/AlgebraicStockFlow.jl.svg?branch=master)](https://travis-ci.com/Xiaoyan-Li/AlgebraicStockFlow.jl)
-[![Build Status](https://ci.appveyor.com/api/projects/status/github/Xiaoyan-Li/AlgebraicStockFlow.jl?svg=true)](https://ci.appveyor.com/project/Xiaoyan-Li/AlgebraicStockFlow-jl)
-[![Coverage](https://codecov.io/gh/Xiaoyan-Li/AlgebraicStockFlow.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/Xiaoyan-Li/AlgebraicStockFlow.jl)
-[![Coverage](https://coveralls.io/repos/github/Xiaoyan-Li/AlgebraicStockFlow.jl/badge.svg?branch=master)](https://coveralls.io/github/Xiaoyan-Li/AlgebraicStockFlow.jl?branch=master)
+[![Tests](https://github.com/AlgebraicJulia/StockFlow.jl/actions/workflows/tests.yml/badge.svg)](https://github.com/AlgebraicJulia/StockFlow.jl/actions/workflows/tests.yml)
+<!-- TODO: Set up on codecov.io for repo [![codecov](https://codecov.io/github/AlgebraicJulia/StockFlow.jl/branch/maaster/graph/badge.svg)](https://app.codecov.io/github/AlgebraicJulia/StockFlow.jl) -->
```

Comment:
@Xiaoyan-Li If you'd like Code Coverage, we can uncomment this line once you've set it up. https://docs.codecov.com/docs